lib/colorado_lottery.rb

```diff
@@ -0,0 +1,50 @@
+
+
+class ColoradoLottery
+  attr_reader :registered_contestants, :winners, :current_contestants
+
+  def initialize
+    @registered_contestants = {}
+    @winners = []
+    @current_contestants = {}
+  end
+
+  def find_game_interest(name, game_interest)
+    name.game_interests.find do |game|
+        if game_interest.name == game
+         return true
+       else
+         return false
+        end
+    end
+  end
```

Comment:
This is not the correct use of `find`.  `find` should be used to get one element from an array, based on a condition - not a true/false based on a condition. Also, because you are returning `false` on the else, this will only ever look at the *first* game_interest - that will either kick out true or false and will never move on the second game interest. You should either:

use `each`:
```ruby
name.game_interests.each do |game|
  if game_interest.name == game
    return true
  end
end
```

or `any?`
```ruby
name.game_interests.any? do |game|
  game_interest.name == game
end
```

or, `include?`
```ruby
name.game_interests.include?(game_interest)
```